README.md

```diff
@@ -138,3 +158,99 @@ HTTP Response:
     "count": int
 }
 ```
+
+## Adding Alarm Metric Log
+
+POST "/metrics/alarm"
+
+Request Body:
+
+```
+{
+    "alarmInfo": [
+        {
+            "username": string,
+            "startTime": string, // get UTC string with: new Date().toUTCString()
+            "endTime": string // get UTC string with: new Date().toUTCString()
+        }
+        ...
+    ]
+}
+```
+
+HTTP Response:
+
+```
+{
+    "addedAlarm": [
+        {
+            "alarmID": integer // keep returned id for future log updates
+        }
+        ...
+    ]
+}
+```
+
+## Update Alarm Log Sent Status
+
+PUT "/metrics/alarm"
+
+Request Body:
+
+```
+{
+    "alarmInfo": [
+        {
+            "alarmID": integer,
```

Comment:
alarmId should be a path param. It identifies a resource, not sort/filter

Reply:
Is there a reason why you put alarmInfo wrapping the actual body data that we are interested in?